src/transpilation/index.ts

```diff
@@ -55,6 +55,16 @@ export function createVirtualProgram(input: Record<string, string>, options: Com
                 return ts.createSourceFile(filename, input[filename], ts.ScriptTarget.Latest, false);
             }
 
+            if (filename.includes("typescript-to-lua")) {
+                const filePath = path.resolve(__dirname, "../../helpers/multi.ts");
+                if (libCache[filename]) return libCache[filename];
+                const content = fs.readFileSync(filePath, "utf8");
+
+                libCache[filename] = ts.createSourceFile(filePath, content, ts.ScriptTarget.Latest, false);
+
+                return libCache[filename];
+            }
+
             if (filename.startsWith("lib.")) {
                 if (libCache[filename]) return libCache[filename];
                 const typeScriptDir = path.dirname(require.resolve("typescript"));
```

Comment:
```ts
            let filePath: string | undefined;

            if (filename.startsWith("lib.")) {
                const typeScriptDir = path.dirname(require.resolve("typescript"));
                filePath = path.join(typeScriptDir, filename);
            }

            if (filename.includes("typescript-to-lua")) {
                filePath = path.resolve(__dirname, "../../helpers/multi.ts");
            }

            if (filePath !== undefined) {
                if (libCache[filename]) return libCache[filename];
                const content = fs.readFileSync(filePath, "utf8");
                libCache[filename] = ts.createSourceFile(filename, content, ts.ScriptTarget.Latest, false);
                return libCache[filename];
            }
```

Also, `filename.includes("typescript-to-lua")` sounds a bit unsafe, is `filename` always relative (if it could be absolute it might fail if someone cloned the repo somewhere in the `typescript-to-lua` directory)